pkg/kubelet/cm/deviceplugin/pod_devices.go

```diff
@@ -252,6 +255,20 @@ func (pdev podDevices) deviceRunContainerOptions(podUID, contName string) *Devic
 				SELinuxRelabel: false,
 			})
 		}
+
+		// Updates for Annotations
+		for k, v := range resp.Annotations {
+			if e, ok := annotationsMap[k]; ok {
+				glog.V(3).Infof("skip existing annotation %s %s", k, v)
```

Comment:
Can we also downgrade it to V(4) here (as the same of add below)? User may not expect to see a lot 'skip' while no 'add'.

Reply:
Done